armi/physics/neutronics/settings.py

```diff
@@ -185,7 +185,7 @@ def defineSettings():
             CONF_EPS_EIG,
             default=1e-07,
             label="Eigenvalue Epsilon",
-            description="convergence criterion for calculating the eigenvalue",
+            description="Convergence criteria for calculating the eigenvalue in the global flux solver",
```

Comment:
Technically it is singular criterion :P

Reply:
I agree with that! But all the other criterion are listed as criteria as well. Plus I looked it up:
https://www.merriam-webster.com/words-at-play/criteria-vs-criterion-singular-plural-grammar

So I just changed the word for consistency.